scheduler/src/cook/kubernetes/api.clj

```diff
@@ -275,7 +275,8 @@
           node-name (some-> node .getMetadata .getName)
           pods-on-node (num-pods-on-node node-name pods)
           labels-on-node (or (some-> node .getMetadata .getLabels) {})
-          matching-node-blocklist-labels (some #(get labels-on-node %) node-blocklist-labels)]
+          matching-node-blocklist-labels (filter #(contains? labels-on-node %) node-blocklist-labels)
+          matching-node-blocklist-keyvals (map (fn [key] [key (get labels-on-node key)]) matching-node-blocklist-labels)]
```

Comment:
Seems like we can simplify this by using [`select-keys`](https://clojuredocs.org/clojure.core/select-keys).

Reply:
Thats a nice function I was previously unaware! Thanks!